Source/sound.cpp

```diff
@@ -468,36 +441,32 @@ void __fastcall music_start(int nTrack)
 }
 // 4A22D4: using guessed type char gbMusicOn;
 
-void __fastcall sound_disable_music(bool disable)
+void __fastcall sound_disable_music(BOOL disable)
 {
-	if ( disable )
-	{
+	if ( disable ) {
 		music_stop();
-	}
-	else if ( sgnMusicTrack != 6 )
-	{
+	} else if ( sgnMusicTrack != 6 ) {

```

Comment:
Looking at the _music_id enum, looks like there should be "TMUSIC_NONE = 6" instead of "NUM_MUSIC = 6" and it should be used here (and above in music_stop)

Reply:
Not sure about this one, the other values of that enum dosn't seam to add up here. But maybe sgnMusicTrack[NUM_MUSIC] would make sens